raster_analysis/data_environment.py

```diff
@@ -209,3 +218,13 @@ def has_default_value(self, name):
         """
         encoding = self.get_pixel_encoding(name)
         return isinstance(encoding, defaultdict) or 0 in encoding
+
+    def dict(self, *args, **kwargs):
+        layers = super(DataEnvironment, self).dict()["layers"]
```

Comment:
BTW, you don't need to specify the parent class and self anymore! :)
https://stackoverflow.com/a/10483204